.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2.1
+
+executors:
+  test_executor:
+    working_directory: ~/sitemaps
+
+    docker:
+      - image: circleci/ruby:${RUBY_VERSION}
+
+jobs:
+  build:
+    environment:
+      RUBY_VERSION: << parameters.ruby_version >>
+    executor: test_executor
+    parameters:
+      ruby_version:
+        type: string
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - sitemaps-cache-v1-{{ arch }}-{{ .Branch }}-{{ .Revision }}
+            - sitemaps-cache-v1-{{ arch }}-{{ .Branch }}
+            - sitemaps-cache-v1
+
+      - run:
+          name: Bundle Install
+          command: |
+            bundle check --path=vendor/bundle || bundle install --clean --path vendor/bundle
+
+      - save_cache:
+          key: sitemaps-cache-v1-{{ arch }}-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - vendor/bundle
+
+      - run:
+          name: RSpec
+          command: bundle exec rspec spec
+
+      # Once lower-level offenses are resolved, the fail-level flags should be removed
+      # so that any offense will cause the build to fail.
+      - run:
```

Comment:
@peggles2 , there's plenty of old code in here that is generating rubocop offenses, so I'm grandfathering it in by setting Rubocop to fail only at offenses that are warning level or above. We'll need to clean up those offenses as we go, then loosen the `fail-level`. We can't set this repo to run on CodeClimate, but this will have a similar effect of blocking builds with Rubocop errors.